api/db-cdr/generate-cdr/generate-cb-criteria-tables.sh

```diff
@@ -39,7 +39,7 @@ then
 fi
 
 # Check that bq_project exists and exit if not
-datasets=$(bq --project=$BQ_PROJECT ls --max_results=150)
+datasets=$(bq --project=$BQ_PROJECT ls --max_results=1000)
```

Comment:
Updated the max results. We probably need to look at purging some dataset in the near future and how to automate it.